tests/e2e/medic-conf-tests.js

```diff
@@ -0,0 +1,45 @@
+const _ = require('lodash');
+const {expect} = require('chai');
+const util = require('util');
+const { API_PORT }=require('../constants');
+const exec = util.promisify(require('child_process').exec);
+
+const actions = ['compile-app-settings','backup-app-settings','upload-app-settings','convert-app-forms',
+  'convert-collect-forms','convert-contact-forms', 'backup-all-forms','upload-app-forms','upload-collect-forms',
+  'upload-contact-forms', 'upload-resources','upload-branding','upload-partners','upload-custom-translations',
+  'upload-privacy-policies', 'delete-all-forms'];
+
+describe('medic-conf supported actions', () => {
+  beforeAll( () =>{
+    // Change the directory
+    try {
+      process.chdir('config/default');
+    } catch (err) {
+      console.error('no need to change directory');
+    }
+  });
+
+  afterAll(() =>{
+    // Change the directory back to cht-core
+    process.chdir('../..');
+  });
+
+  const runCommand = async (action) => {
+    const { stdout, stderr } = await exec(`medic-conf --url=http://admin:pass@localhost:${API_PORT} ${action} --force`);
```

Comment:
I think you can also build this url by getting the rest of the available variables in the `constants.js` and `auth.js`. Have a look at the `conf.js`

 `http://${auth.username}:${auth.password}@${constants.API_HOST}:${constants.API_PORT}`